doc/source/tutorials-website/create_tutorials_html.py

```diff
@@ -18,7 +18,9 @@ def __init__(self, path, title):
         self.title = title
         # set url and update from markdown to ipynb
         self.url = url_prefix + self.path.replace(".md", ".ipynb")
-
+        self.url = self.url.replace("/var/folders/n5/lj0kjnmd4rx1_q9yv2m0ygd80000gn/T", "")
+        self.url = re.sub(r"tmp[a-z0-9_]+/", "", self.url)
+     
```

Comment:
Sill a little bit off the target, the generated link now is 
`https://nbviewer.org/urls/qutip.org/qutip-tutorials/tmp/tutorials-v5/quantum-circuits/quantum-gates.ipynb
`
on my broswer.

Reply:
@BoxiLi My bad, now it's fixed